service/src/test/java/bio/terra/cbas/controllers/VerifyPactsRunSetsApiController.java

```diff
@@ -184,4 +188,48 @@ public void runSetsData() throws Exception {
     when(smartRunSetsPoller.updateRunSets(response))
         .thenReturn(new TimeLimitedUpdater.UpdateResult<>(response, 1, 1, true));
   }
+
+  @State({"a run set with UUID 20000000-0000-0000-0000-000000000002 exists"})
+  public void postAbort() throws Exception {
+    UUID runSetId = UUID.fromString("20000000-0000-0000-0000-000000000002");
+    UUID runId = UUID.fromString("30000000-0000-0000-0000-000000000003");
+    RunSet runSetToBeCancelled =
+        new RunSet(
+            runSetId,
+            null,
+            null,
+            null,
+            null,
+            CbasRunSetStatus.RUNNING,
+            null,
+            null,
+            null,
+            1,
+            null,
+            null,
+            null,
+            null);
+
+    Run runToBeCancelled =
+        new Run(
+            runId,
+            UUID.randomUUID().toString(),
+            runSetToBeCancelled,
+            null,
+            null,
+            CbasRunStatus.RUNNING,
+            null,
+            null,
+            null);
+
+    AbortRequestDetails abortDetails = new AbortRequestDetails();
+    abortDetails.setFailedIds(List.of());
+    abortDetails.setSubmittedIds(List.of(runToBeCancelled.runId()));
+
+    when(runSetDao.getRunSet(runSetId)).thenReturn(runSetToBeCancelled);
+    when(runDao.getRuns(new RunDao.RunsFilters(runSetId, any())))
+        .thenReturn(Collections.singletonList(runToBeCancelled));
```

Comment:
TOL: If these are always returning the "to be cancelled" version of the run and run set, then what sets the status to `CANCELING` to satisfy the expected response on the client side of the pact?